mat_dp_pipeline/sdf/standard_data_format.py

```diff
@@ -8,25 +8,20 @@
 import numpy as np
 import pandas as pd
 
-from mat_dp_pipeline.common import FileOrPath
+from mat_dp_pipeline.common import FileOrPath, validate
 
 Year = int
 
 
-def validate(condition: bool, error_message: str | None = None):
-    if not condition:
-        raise ValueError(error_message)
-
-
-def validate_tech_units(tech_meta: pd.DataFrame):
-    if tech_meta.empty:
+def validate_tech_units(tech_metadata: pd.DataFrame) -> None:
+    if tech_metadata.empty:
         return
     validate(
-        (tech_meta.loc[:, "Material Unit"].groupby(level=0).nunique() == 1).all(),
+        (tech_metadata.loc[:, "Material Unit"].groupby(level=0).nunique() == 1).all(),
         "There are tech categories with non-unique Material Unit!",
     )
     validate(
-        (tech_meta.loc[:, "Production Unit"].groupby(level=0).nunique() == 1).all(),
+        (tech_metadata.loc[:, "Production Unit"].groupby(level=0).nunique() == 1).all(),
         "There are tech categories with non-unique Production Unit!",
```

Comment:
I think what bothers me about validate is that the message does the opposite of the condition. `.all() => There *are* techs...`
Here I think it's more readable than below with validate, but just a general comment

Reply:
To drop any further discussions about `validate`: do you want to remove this function? If yes I'm going to do it.

Reply:
Yeah - let's remove then, just make sure to update the bools to the opposite

Reply:
(e.g. .any() etc)

Reply:
Sure.